### DISCUSSION TITLE
Free hand selection using mouse

### DISCUSSION THREAD

**Discussion by sachindongare029** [konvajs/konva#1208] [Q&A]
Hi all,
   I want to implement a free hand area selection on canvas using react-konva. Anyone have solution or suggestion ? Please refer screenshot for better understanding.
   Thanks!!!
![Screenshot from 2021-11-01 14-06-49](https://user-images.githubusercontent.com/40762204/139645284-327e9033-c75a-4562-893a-5b403ef7faa7.png)

**Comment by lavrton** [comment]
Just for the reference: https://stackoverflow.com/questions/69795950/react-konva-free-hand-area-selection-using-mouse-click-straight-lines
